README.yaml

```diff
@@ -0,0 +1,95 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: terraform-aws-user-data-dns-proxy
+
+# Tags of this project
+tags:
+  - aws
+  - terraform
+  - terraform-modules
+  - user-data
+  - cloud
+
+# Categories of this project
+categories:
+  - terraform-modules/user-data
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "APACHE2"
+
+# Canonical GitHub repo
+github_repo: cloudposse/terraform-aws-user-data-dns-proxy
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/terraform-aws-user-data-dns-proxy.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/terraform-aws-user-data-dns-proxy"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/terraform-aws-user-data-dns-proxy.svg"
+    url: "https://github.com/cloudposse/terraform-aws-user-data-dns-proxy/releases/latest"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+related:
+  - name: "terraform-template-user-data-assests"
+    description: "terraform-template-user-data-assests"
+    url: "https://travis-ci.org/cloudposse/terraform-template-user-data-assests"
```

Comment:
Wrong related URLs. These will never be travis.

Reply:
done